tests/rptest/tests/flink_basic_test.py

```diff
@@ -214,14 +229,39 @@ def test_transaction_workload(self):
         #     /workloads/data/part-cfbe2720-7117-40fd-8c28-fca31a459ff8-0-0
         # Data sample:
         # ...
-        # 201,"2024-01-05 20:38:53.125",46
+        # 542,415,31,"2024-01-09 22:22:43.647",17,TiJJtOuzOkiOsGmws
```

Comment:
I think this would OOM when run in a scale setting, we ran into similar issues before, we had to reimplement some sorta streaming verifier.

I think we have to write some deterministic data into the topic (eg: monotonic sequence of integers or some such) so verification doesn't need copying and loading these huge files in memory.

Reply:
This is a simple test and supposed to have lesser number in it. Implementing the rolling checker will be a part of the scale activity.